third_party/cpgfunctionEP-0.3.2/CHANGELOG.md

```diff
@@ -0,0 +1,211 @@
+# History of changes - cpgfunctionEP
```

Comment:
A history of changes (CHANGELOG) for cpgfunctionEP and the project prior to a fork, cpgfunction.